packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -24,35 +28,23 @@ export default {
 		size: { default: null, type: String },
 		type: { default: null, type: String },
 	},
-	data() {
+	setup(props, { emit, slots }) {
+		const hasSlot = computed(() => Boolean(slots.default))
 		return {
-			isHover: false,
+			handleClick: (event) => emit(KottiButton.Event.CLICK, event),
+			hasSlot,
+			mainClasses: computed(() => ({
+				'kt-button': true,
+				[props.type]: true,
+				icon: props.icon,
+				'icon-only': props.icon && !hasSlot.value && !props.label,
+				'kt-button--is-block': props.isBlock,
+				'kt-button--is-multiline': props.isMultiline,
+				sm: props.size === 'small',
```

Comment:
not on you but
i think we pass `sm`, `md`, `lg` to the `size` prop and apparently we only either `sm` or not

would propose 
`[props.size]: true`

Reply:
hmmm , [documentation](https://3yourmind.github.io/kotti/usage/components/button) claims differently. Will have a look